### PR TITLE
Added ignoreChunks config option

### PR DESCRIPTION
**Pull request by aweber1**
**TL;DR**

Added an `ignoreChunks` config option to the plugin. This allows config to be:

```
new WebPackSplitByPathPlugin(
  [
    {
      name: "vendor",
      path: path.join(__dirname, "../node_modules")
    }
  ],
  {
    ignoreChunks: ["main"]
  }
)
```

And any chunk named `main` will be ignored by the split-by-path plugin.

**Longer explanation**

I encountered an edge case when attempting to use web workers and the webpack worker-loader (https://github.com/webpack/worker-loader) along with the webpack-split-by-path plugin.

Given this configuration for the plugin:

```
new WebPackSplitByPathPlugin(
      [
        {
          name: "vendor",
          path: path.join(__dirname, "../node_modules")
        }
      ])
```

My code bundle is split into "app.js" and "vendor.js" as expected. 

I also have a web worker in the code base that imports modules from node_modules. Using the webpack worker-loader means the web worker gets bundled as a separate file - as expected. However, because the web worker imports from node_modules, the split by path plugin was also splitting the web worker bundle. So instead of a single `main.worker.js`, I was getting `main.worker.js` and `vendor.worker.js`.

While this is actually pretty cool behavior, unfortunately only one of the worker files was getting loaded. And while that file did attempt to load the other worker file via `importScripts`, the other file wasn't loading. I suspect this could have been handled by modifying the worker-loader, but it was much easier to modify the split-by-path plugin to simply ignore the worker chunk altogether.

By adding an `ignoreChunk` config option and specifying the name of the web worker chunk, I now get a single worker bundle `main.worker.js` and everything loads as expected.

### CODE REVIEW

**Comment by BohdanTkachenko** [comment]
Thanks for the PR. Merged and published version 0.0.10 to NPM.

**Comment by e-cloud** [comment]
This changes are supposed to be mentioned in README.md.
